Makefile

```diff
@@ -48,22 +48,21 @@ unit-test: generate fmt vet
 	GO111MODULE=off go list ./... | grep -v controller | xargs go test ${CI_TEST_FLAGS} ${LOCAL_FLAGS}
 
 # Build manager binary
-manager: generate fmt vet
-	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=off  go build -o bin/manager cmd/controller/main.go
+manager: dataset-controller-build alluxioruntime-controller-build csi-build
```

Comment:
Should `make manager` do `csi-build` task? This looks like a `make all` thing

Reply:
Thanks, I will change `manager` to `build`.